app/views/ReportView/components/Summary/index.tsx

```diff
@@ -11,36 +11,43 @@ const Summary = ({
   templateName,
   ...props
 }: SummaryProps): JSX.Element => {
-  if (templateName === 'probe') {
+  // TODO remove backup template name checks when data is updateed in prod
+  const summarySection = props.visibleSections.find(element => ['summary-genomic', 'summary-tgr', 'summary-pcp', 'summary-probe'].includes(element));
```

Comment:
I got an error with this line when trying to print the report: "Cannot read properties of undefined (reading 'find').
TypeError: Cannot read properties of undefined (reading 'find')"

Reply:
fixed, had to update the SummaryProps type